File: VS Code Projects/Java/Pallindrome.java
import java.util.*;
public class Pallindrome {
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter a Number : ");
        int n=sc.nextInt();
        int t=n, r=0, rev=0;
        while(t>0)
        {
            r=t%10;
            rev=(rev*10)+r;
            t=t/10;
        }
        if(rev==n)
        {
            System.out.println("Pallindrome");
        }
        else 
        {
            System.out.println("Not Pallindrome");
        }
    }
}

File: VS Code Projects/C/pallindrome.c
#include <stdio.h>
int main()
{
    int n, rev, r, t;
    printf("Enter Number : %d", n);
    scanf("%d", &n);
    t = n;
    while (t > 0)
    {
        r = t % 10;
        rev = rev * 10 + r;
        t = t / 10;
    }
    if (rev == n)
    {
        printf("Pallindrome");
    }
    else
    {
        printf("Not Pallindrome");
    }
    return 0;
}